meta-resin-common/classes/image-resin.bbclass

```diff
@@ -8,12 +8,16 @@ inherit image_types_resin
 # and ship them in the deploy directory for out-of-tree kernel modules build
 DEPENDS += "kernel-modules-headers"
 
-# Deploy license.manifest
-DEPLOY_IMAGE_LICENSE_MANIFEST = "${DEPLOY_DIR_IMAGE}/${IMAGE_NAME}.rootfs.license.manifest"
-DEPLOY_SYMLINK_IMAGE_LICENSE_MANIFEST = "${DEPLOY_DIR_IMAGE}/${IMAGE_LINK_NAME}.license.manifest"
-IMAGE_LICENSE_MANIFEST = "${LICENSE_DIRECTORY}/${IMAGE_NAME}/license.manifest"
 # Deploy the license.manifest of the current image we baked
 deploy_image_license_manifest () {
+    IMAGE_LICENSE_MANIFEST="${LICENSE_DIRECTORY}/${IMAGE_NAME}/license.manifest"
+    if [ -d "${IMGDEPLOYDIR}" ]; then
```

Comment:
Is this only used for old versions of poky? If so a comment to that effect would be useful to allow it to be removed in future.